libs/designsystem/dropdown/src/dropdown.module.ts

```diff
@@ -22,9 +23,10 @@ const declarations = [DropdownComponent];
     PopoverComponent,
     ButtonComponent,
     CommonModule,
+    ListItemTemplateDirective,
```

Comment:
Consider leaning into the current module-based approach (until we get stuff properly converted to standalone) by importing the `ListModule` and still exporting `ListItemTemplateDirective`, but without making it standalone